integrator/integrator.py

```diff
@@ -0,0 +1,98 @@
+from sys import argv
+from subprocess import Popen
+import os
+from integratorlib import *
+
+
+# Pack size for integrator
```

Comment:
All code should be moved into the functions. There is too much code out of the function that makes the testing very difficult. 

Use the following rules (they are not mandatory, however, they are useful):

1. Each function should have <= 15 lines of the code.
2. Each file should have <=200 lines of the code.
3. The code out of the functions (except imports and definitions) should have <=15 lines of the code.